starter-code/db/migrate/20151022092530_changes.rb

```diff
@@ -0,0 +1,8 @@
+class Changes < ActiveRecord::Migration
+  def change
+      add_column :managers, :phone_number, :string
+      add_column :songs, :downloads, :string
+      add_column :songs, :artist_last_name, :string
+      rename_column :managers, :phone_number, :cell_phone_number
```

Comment:
should be in a separate migration... otherwise might as well call it `:cell_phone_number` directly no? ;)